std/datetime/systime.d

```diff
@@ -2391,7 +2391,7 @@ public:
             allowOverflow = Whether the days should be allowed to overflow,
                             causing the month to increment.
       +/
-    ref SysTime add(string units)(long value, AllowDayOverflow allowOverflow = AllowDayOverflow.yes) @safe nothrow
+    ref SysTime add(string units)(long value, AllowDayOverflow allowOverflow = AllowDayOverflow.yes) @safe nothrow scope
```

Comment:
I take it that `scope` is supposed to be here, because it returns by `ref`? If that's the case, then it makes more sense to me to put the `scope` on the left-hand side with the `ref`.

Reply:
I checked combinations of omitting scope and it turned out to be dispensable here (but doesn't hurt either; seems to be inferred), but adjTime require it. See my comment there.

Reply:
Please don't add `scope` if it's not required, especially since the DIP 1000 implementation isn't completely sorted out.

Reply:
@jmdavis  done: Removed scope here